docs/sce_file_contents.md

```diff
@@ -296,3 +303,161 @@ For methods that rely on the HTO data, these statistics are found in the `colDat
+
+Metadata associated with {ref}`data processing <processing_information:Processing information>` is included in the `.uns` slot as a list.
+
+```python
+adata_object.uns # experiment metadata
+```
+
+All of the object metadata included in `SingleCellExperiment` objects are present in the `.uns` slot of the `AnnData` object.
+To see a full description of the included columns, see the [section on experiment metadata in `Components of a SingleCellExperiment object`](#singlecellexperiment-experiment-metadata).
+The only exception is that the `AnnData` object _does not_ contain the `sample_metadata` item in the `.uns` slot.
+Instead, the contents of the `sample_metadata` data frame are stored in the cell-level metadata (`.obs`).
+
+The `AnnData` object also includes the following additional items in the `.uns` slot:
+
+| Item name   | Contents                                                         |
+| ------------- | ---------------------------------------------------------------- |
+| `schema_version` | CZI schema version used for `AnnData` formatting, `3.0.0` |
+
+
+### `AnnData` dimensionality reduction results
+
+In the HDF5 file containing the processed `SingleCellExperiment` object only (`_processed_rna.hdf5`), the `.obsm` slot of the object will be occupied with both principal component analysis (`X_PCA`) and `X_UMAP` results.
+For all other HDF5 files, the `.obsm` slot will be empty as no dimensionality reduction was performed.
+
+For information on how PCA and UMAP results were calculated see the {ref}`section on processed gene expression data <processing_information:Processed gene expression data>`.
+
+The following command can be used to access the PCA and UMAP results:
+
+```python
+adata_object.obsm["X_PCA"] # pca results
+adata_object.obsm["X_UMAP"] # umap results
+```
+
+### Additional `AnnData` components for CITE-seq libraries (with ADT tags)
+
+ADT data from CITE-seq experiments, when present, is available as a separate `AnnData` object (`.hdf5` file).
+All files containing ADT data will contain the `_adt.hdf5` suffix.
+
+The data matrix, `X`, of the `AnnData` objects contain the primary ADT expression data as integer counts in both the unfiltered (`_unfiltered_rna.hdf5`) and filtered (`_filtered_rna.hdf5`) objects.
+Each column corresponds to a cell or droplet (in the same order as the main `AnnData` object) and each row corresponds to an antibody derived tag (ADT).
+Column names are again cell barcode sequences and row names are the antibody targets for each ADT.
+
+As with the RNA `AnnData` objects, in processed objects _only_ (`_processed_adt.hdf5`), the data matrix `X` contains the normalized ADT counts and the primary data can be found in `raw.X`.
+Note that only cells which are denoted as "Keep" in  the `adata_obj.uns["adt_scpca_filter"]` column (as described [above](#singlecellexperiment-cell-metrics)) have normalized expression values in the `logcounts` assay, and all other cells are assigned `NA` values.
```

Comment:
```suggestion
Note that only cells which are denoted as `"Keep"` in the `adata_obj.uns["adt_scpca_filter"]` column (as described [above](#singlecellexperiment-cell-metrics)) have normalized expression values in the `logcounts` assay, and all other cells are assigned `NA` values.
```

Reply:
**BUT** - we're in anndata land so there is no `logcounts` assay, so this text needs to change beyond my suggestion